apps/backoffice-v2/src/pages/TransactionMonitoringAlertsAnalysis/hooks/useTransactionMonitoringAlertsAnalysisPageLogic/useTransactionMonitoringAlertsAnalysisPageLogic.tsx

```diff
@@ -30,12 +30,14 @@ export const useTransactionMonitoringAlertsAnalysisPageLogic = () => {
     }
 
     navigateBack();
+    navigateBack();
```

Comment:
You should probably leave a comment here explaining the duplication

Reply:
This change should be reverted.